plugins/plugin-docker/che-plugin-openshift-client/src/main/java/org/eclipse/che/plugin/openshift/client/OpenShiftConnector.java

```diff
@@ -315,6 +316,7 @@ public ContainerCreated createContainer(CreateContainerParams createContainerPar
                                                  .withName(imageStreamName)
                                                  .get();
         if (imageStream == null) {
+            openShiftClient.close();
```

Comment:
I agree that it's not a great solution. Wrapping the whole method body in a try causes some variables used in the catch to go out of scope. Instead, I've separated out the two distinct uses of OpenShiftClient into two separate try blocks. WDYT?